### PR TITLE
W-14809904 update create pdk project gr

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: pdk/1.0/modules/ROOT/pages/policies-pdk-create-project.adoc [hunk 1]
```diff
@@ -81,7 +72,8 @@ After running the setup command, the policy's root directory contains the follow
 │  └── gcl.yaml # GCL file managed by the developer that describes the Policy Definition
 ├─ target/      # Contains the implementation asset files and the output of the policy build
 ├─ src/         # Contains the implementation source code
-├─ test/        # Contains the artifacts required for running the policy locally
+├─ playground/  # Contains the artifacts required for running the policy locally
+├─ test/        # Contains the source and configuration files to write and run integration tests
```
Comment (nledesma) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-create-project.adoc]: This folder is named `tests/` not `test/`

**Review by nledesma — APPROVED** [review]

File: pdk/1.0/modules/ROOT/pages/policies-pdk-create-project.adoc [hunk 2]
```diff
@@ -26,6 +28,8 @@ To create a custom policy project, run the PDK CLI `create` command, replacing `
 anypoint-cli-v4 pdk policy-project create --name <my-custom-policy>
 ----
 
+NOTE: If you receive a prompt requesting your group ID, see xref:policies-pdk-troubleshooting.adoc#group-id-error[].
```
Comment (bjoson-mulesoft) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-create-project.adoc]: In the past, if you linked to an anchor like this, you'd need to include the title in the []. Is this no longer the case?

File: pdk/1.0/modules/ROOT/pages/policies-pdk-create-project.adoc [hunk 3]
```diff
@@ -43,34 +47,21 @@ The `create` command supports the following flags:
 If the directory does not exist, it is created. If no directory is specified, the directory name is the policy `assetID`.
 | `--version=<value>` | `1.0.0` | No | Version of the new policy. +
 The version format is `<major-release>.<minor-release>.<patch-release>`.
+| `--group-id=<value>` | N/A | No | Organization ID of the organization that owns the policy. +
+For additional information about this flag, see xref:policies-pdk-troubleshooting.adoc#group-id-error[].
```
Comment (bjoson-mulesoft) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-create-project.adoc]: Same comment as the previous one about linking to an anchor

File: pdk/1.0/modules/ROOT/pages/policies-pdk-create-project.adoc [hunk 4]
```diff
@@ -92,9 +84,9 @@ After running the setup command, the policy's root directory contains the follow
 There are two separate `target` directories:
 
 * `definition/target/`: Contains the policy asset definition published to Exchange.
-* `target/`: Contains the files for the Policy asset implementation and the output of the Rust compilation process.
+* `target/`: Contains the files for the policy's Exchange asset implementation and the output of the project build
```
Comment (bjoson-mulesoft) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-create-project.adoc]: Should this end with a period?

File: pdk/1.0/modules/ROOT/pages/policies-pdk-create-project.adoc [hunk 5]
```diff
@@ -92,9 +84,9 @@ After running the setup command, the policy's root directory contains the follow
...
 
-Both `target` directories and their content are included in the `.gitignore` file. You must compile the targets on each individual devices. For example, if you are developing a policy as part of a team, your team can publish the policy and it's source code to a remote repository, but each team member must build the target on their device for local testing.
+Both target directories and their content are included in the .gitignore file. You must compile the targets on each individual devices using the `make build` command. For example, if you develop a policy as part of the team, your team can publish the policy and its source code to a remote repository, but each team member must build the target on their device to work with the policy.
```
Comment (bjoson-mulesoft) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-create-project.adoc]: I'm curious why you removed the formatting from .gitignore? According to our standards, files names should be in mono font: https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#:~:text=a%20faull%20filename.-,font%20choices,-Follow%20these%20rules

**Review by bjoson-mulesoft — APPROVED** [review]

File: pdk/1.0/modules/ROOT/pages/policies-pdk-create-project.adoc [hunk 6]
```diff
@@ -43,34 +47,21 @@ The `create` command supports the following flags:
 If the directory does not exist, it is created. If no directory is specified, the directory name is the policy `assetID`.
 | `--version=<value>` | `1.0.0` | No | Version of the new policy. +
 The version format is `<major-release>.<minor-release>.<patch-release>`.
+| `--group-id=<value>` | N/A | No | Organization ID of the organization that owns the policy. +
+For additional information about this flag, see xref:policies-pdk-troubleshooting.adoc#group-id-error[Troubleshooting Group ID]].
```
Comment (bjoson-mulesoft) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-create-project.adoc]:
```suggestion
For additional information about this flag, see xref:policies-pdk-troubleshooting.adoc#group-id-error[Troubleshooting Group ID].
```

**Review by bjoson-mulesoft — APPROVED** [review]

**Pull request merged**